### ISSUE TITLE
Converter from AT to ET

### ISSUE THREAD

**Issue by Firesoul7** [PhelanBavaria/ancienttimeline#59]
For initial release we should have an (at least mostly) working converter from AT to ET. Personally I do not know how to achieve this. I think that building this converter will be up to Qweytr.

**Comment by Science-Recon** [comment]
Yes, I'm not sure how we want to/will go about it and I dont know anything on there than python, though I'd be happy to lear something different to do it.

**Comment by qweytr** [comment]
It has been my plan to do it myself all along. It will be coded in python and it will work by creating a new mod that is then run on top of ET.
